TradeItIosTicketSDK2/TradeItOrdersTableViewManager.swift

```diff
@@ -0,0 +1,240 @@
+            buildOrderSectionPresentersFrom(splitedOrdersArray: splitedOtherOrdersArray)
+        }
+        
+        self.ordersTable?.reloadData()
+    }
+    
+    // MARK: UITableViewDelegate
+    
+    func tableView(_ tableView: UITableView, titleForHeaderInSection section: Int) -> String? {
+        return self.orderSectionPresenters[section].title
+    }
+    
+    // MARK: UITableViewDataSource
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        return self.orderSectionPresenters[indexPath.section].cell(forTableView: tableView, andRow: indexPath.row)
+    }
+    
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        guard let orderSectionPresenter = self.orderSectionPresenters[safe: section] else { return 0 }
+        return orderSectionPresenter.numberOfRows()
+    }
+    
+    func numberOfSections(in tableView: UITableView) -> Int {
+        if self.orderSectionPresenters.isEmpty {
+            self.ordersTable?.backgroundView = noResultsBackgroundView
+        } else {
+            self.ordersTable?.backgroundView = nil
+        }
+        return self.orderSectionPresenters.count
+    }
+    
+    func tableView(_ tableView: UITableView, estimatedHeightForRowAt indexPath: IndexPath) -> CGFloat {
+        return CGFloat(TradeItOrdersTableViewManager.ORDER_CELL_HEIGHT)
+    }
+    
+    func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
+        return CGFloat(TradeItOrdersTableViewManager.ORDER_CELL_HEIGHT)
+    }
+    
+    func tableView(_ tableView: UITableView, viewForHeaderInSection section: Int) -> UIView? {
+        return self.orderSectionPresenters[section].header(forTableView: tableView)
+    }
+    
+    func tableView(_ tableView: UITableView, heightForHeaderInSection section: Int) -> CGFloat {
+        return self.orderSectionPresenters[section].heightForHeaderInSection()
+    }
+    
+    func tableView(_ tableView: UITableView, heightForFooterInSection section: Int) -> CGFloat {
+        return CGFloat.leastNormalMagnitude
+    }
+    
+    // MARK: Private
+    
+    private func addRefreshControl(toTableView tableView: UITableView) {
+        let refreshControl = UIRefreshControl()
+        refreshControl.attributedTitle = NSAttributedString(string: "Refreshing...")
+        refreshControl.addTarget(
+            self,
+            action: #selector(initiateRefresh),
+            for: UIControlEvents.valueChanged
+        )
+        TradeItThemeConfigurator.configure(view: refreshControl)
+        tableView.addSubview(refreshControl)
+        self.refreshControl = refreshControl
+    }
+    
+    /**
+     * This is to split orders in order to have a specific section for group orders
+    **/
+    private func getSplittedOrdersArray(orders: [TradeItOrderStatusDetails]) -> [[TradeItOrderStatusDetails]]{
```

Comment:
Is there a better name we can think of to use here? I'm not 100% sure what it's doing by something like `groupOrderLegs` or `collateOrders`?